pymodbus/transport/transport.py

```diff
@@ -156,17 +156,21 @@ def __init__(
         self.is_server = is_server
         self.is_closing = False
 
-        self.transport: asyncio.BaseTransport = None
-        self.loop: asyncio.AbstractEventLoop = None
+        self.loop: asyncio.AbstractEventLoop = None  # type: ignore[assignment]
```

Comment:
This is why I do not like the "| None", sometimes you use it, other times not...that its not really intuitive !

In my mind mypy is a bit stupid in this respect !

Reply:
This one was quite difficult because I could not figure out the intended usage of `get_running_loop()`.